cluster-configuration-example/services-configuration.yaml

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+cluster:
+
+  # Choose proper nvidia driver version from this url http://www.nvidia.com/object/linux-amd64-display-archive.html
+  nvidia_drivers_version: 384.111
+
+  # static docker-version
+  # https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz
+  # Docker client used by hadoop NM (node manager) to launch Docker containers (e.g., of a deep learning job) in the host env.
+  dockerverison: 17.06.2
+
+  # HDFS, zookeeper data path on your cluster machine.
+  dataPath: "/datastorage"
```

Comment:
>dataPath [](start = 2, length = 8)

I saw Hungarian notations like "dataPath", separated with "-", separated with "_", or without any separation such as "imagelist". Can we unify the ways to separate words?